src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqPipelineSpark.java

```diff
@@ -28,16 +27,99 @@
 import java.util.Iterator;
 import java.util.List;
 
+/**
+ * Combined tool that performs all PathSeq steps: read filtering, microbe reference alignment and abundance scoring
+ *
+ * <p>PathSeq is a suite of tools for detecting microbial organisms in deep sequenced biological samples. It is capable of
+ * (1) quantifying microbial abundances in metagenomic samples containing a mixture of organisms, (2) detecting extremely
+ * low-abundance (<0.001%) organisms, and (3) identifying unknown sequences that may belong to novel organisms. The pipeline is based
+ * on a previously published tool of the same name (Kostic et al. 2011), which has been
+ * used in a wide range of studies to investigate novel associations between pathogens and human disease.</p>
+ *
+ * <p>The pipeline consists of three phases: (1) removing reads that are low quality, low complexity, or match a given
+ * host (e.g. human) reference, (2) aligning the remaining reads to a microorganism reference, and (3) determining
+ * the taxonomic classification of each read and estimating microbe abundances. These steps can be performed individually using
+ * PathSeqFilterSpark, PathSeqBwaSpark, and PathSeqScoreSpark. To simplify using the pipeline, this tool combines the
+ * three steps into one. Further details can be found in the individual tools' documentations.</p>
+ *
+ * <p>The filtering phase ensures that only high fidelity, non-host reads are classified, thus reducing computational costs
+ * and false positives. Note that while generally applicable to any type of biological sample (e.g. saliva, stool), PathSeq
+ * is particularly efficient for samples containing a high percentage of host reads (e.g. blood, tissue, CSF). PathSeq
+ * is able to detect evidence of low-abundance organisms and scales to use comprehensive genomic database references
+ * (e.g. > 100 Gbp). Lastly, because PathSeq works by identifying both host and known microbial sequences, it can also
+ * be used to discover novel pathogens by deducing the sample to sequences of unknown origin, which may be followed
+ * by de novo assembly.</p>
+ *
+ * <p>Because sequence alignment is computationally burdensome, PathSeq is integrated with Apache Spark,
+ * enabling parallelization of all steps in the pipeline on multi-core workstations and cluster environments. This
+ * overcomes the high computational cost and permits rapid turnaround times (minutes to hours) in deep sequenced samples.</p>
+ *
+ * <h4>Reference files</h4>
+ *
+ * <p>Before running the PathSeq pipeline, the host and microbe references must be built. Recommended pre-built references
+ * are available as <a href="https://software.broadinstitute.org/gatk/download/bundle">resource bundles</a>.</p>
+ *
```

Comment:
Recommended pre-built references --> Prebuilt references for a standard microbial set
are available as resource bundles --> are available in the GATK Resource Bundle.

Reply:
Done